### DISCUSSION TITLE
Common include [GRAMMAR] for several documents (per project)

### DISCUSSION THREAD

**Discussion by elfman2** [strictdoc-project/strictdoc#1094] [Ideas]
actually **[FRAGMENT_FROM_FILE]** does not support **[GRAMMAR]** inclusion
**[GRAMMAR]** is a "project" feature more than a per document.
currently there is no "project" artefact supported by strictdoc.

**Comment by stanislaw** [comment]
This is being included to the short-term roadmap. Promoting this discussion to an issue.

**Comment by stanislaw** [comment]
Closing this as something that was implemented in 2024 Q1/Q2.
